app/components/createContainer/createContainerController.js

```diff
@@ -265,6 +463,11 @@ function ($q, $scope, $state, $stateParams, $filter, Container, ContainerHelper,
     Container.query({}, function (d) {
       var containers = d;
       $scope.runningContainers = containers;
+      if ($stateParams.from !== '') {
+        loadFromContainerSpec();
+      } else {
+        $scope.fromContainer = {};
```

Comment:
Related to the registry update dropdown.

Add `$scope.formValues.Registry = {};` below `$scope.fromContainer = {};`.